cellxgene_resources/curation_qa.ipynb

```diff
@@ -400,46 +400,37 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def TENx_barcode_checker(df, num_to_check):\n",
-    "    if df.shape[0] < num_to_check:\n",
-    "        num_to_check = df.shape[0]\n",
+    "def TENx_barcode_checker(csv, num_to_check=None, barcode_to_check=None):\n",
```

Comment:
should remove the barcode_to_check parameter and dependent code - I gave an ambiguous use case